util/chplenv/__init__.py

```diff
@@ -19,6 +19,7 @@
     'chpl_threads',
     'chpl_timers',
     'chpl_wide_pointers',
+    'utils',
```

Comment:
Oh ok. This is the first time utils is being used outside of the scripts that live in the same directory. Ok, makes sense then.